src/Microsoft.ML.FastTree/Training/DcgPermutationComparer.cs

```diff
@@ -44,10 +44,10 @@ public static DcgPermutationComparer GetDcgPermutationFactory(string name)
     /// </summary>
     public class DescendingStablePessimisticPermutationComparer : DescendingStablePermutationComparer
     {
-#pragma warning disable TLC_GeneralName // The naming is the least of this class's problems. A setter with no getter??
+#pragma warning disable MSML_GeneralName // The naming is the least of this class's problems. A setter with no getter??
```

Comment:
> The naming is the least of this class's problems. A setter with no getter??

:bulb: Comments like this should be in issues, not comments in source code.

Reply:
TBH when it comes to the FastTree project's structuring, I don't even know where to start with its problems.